### PR TITLE
Fix incorrect endpoints for system collections

### PR DESCRIPTION
**Pull request by licitdev**
Follow-up of #18631 on other occurrences of incorrect endpoints for system collections.

### CODE REVIEW

File: app/src/layouts/kanban/kanban.vue [hunk 1]
```diff
@@ -156,6 +156,16 @@ const { t } = useI18n();
...
+function getItemRoute(primaryKey: string | number) {
+	if (!props.collection) return '';
+
+	const route = props.collection.startsWith('directus_')
+		? props.collection.substring(9)
+		: `content/${props.collection}`;
+
+	return `/${route}/${primaryKey}`;
+}
+
```
Comment (paescuj) [inline app/src/layouts/kanban/kanban.vue:168]: Would it make sense to put this into `@directus/utils` as well?

Comment (rijkvanzanten) [reply app/src/layouts/kanban/kanban.vue:168]: Yes!

**Review by rijkvanzanten — APPROVED** [review]

**Review by azrikahar — APPROVED** [review]
Changes makes sense! 👍 just have a couple tweak suggestions.

File: app/src/layouts/kanban/kanban.vue (hunk 1 above)
Comment (licitdev) [reply app/src/layouts/kanban/kanban.vue]: Extracted it as an app util as it is app specific, with routing to `/content`.

**Review by paescuj — APPROVED** [review]
LGTM 👍

**Pull request merged**